karyon2-admin/build.gradle

```diff
@@ -18,8 +18,8 @@ apply plugin: 'osgi'
 dependencies {
     dependencies {
         compile 'javax.ws.rs:jsr311-api:1.1.1'
-        compile 'javax.servlet:servlet-api:2.5'
-        compile 'org.mortbay.jetty:jetty:6.1.26'
+        provided 'javax.servlet:javax.servlet-api:3.0.1'
+        provided 'org.mortbay.jetty:jetty:6.1.26'
```

Comment:
It seems odd to make jetty provided when it is being used as an embedded server. What if the application is running outside of a servlet container or using a container other than jetty such as tomcat? Is the user expected to supply and keep track of the appropriate version of jetty?